x-pack/plugin/ml/src/test/java/org/elasticsearch/xpack/ml/job/NodeLoadTests.java

```diff
@@ -25,6 +25,6 @@ public void testIncrementCounts() {
             .build();
 
         assertThat(nodeLoad.getNumAssignedJobs(), equalTo(6));
-        assertThat(nodeLoad.remainingJobs(), equalTo(4));
+        assertThat(nodeLoad.remainingJobs(), equalTo(7));
```

Comment:
The method names are confusing now.  "Assigned jobs" is the same as before but "remaining jobs" is different, so "jobs" must have a different meaning in two methods of the same class.

Maybe:

* `getNumAssignedJobs` -> `getNumAssignedJobsAndModels`
* `incNumAssignedNativeInferenceJobs` -> `incNumAssignedNativeInferenceModels`